package.json

```diff
@@ -41,24 +41,22 @@
   },
   "scripts": {
     "cache-arc-dev": "ts-node src/bg_cache_worker.ts",
-    "cache-arc-staging": "cross-env NODE_ENV=production arcjs_network=kovan ts-node src/bg_cache_worker.ts",
-    "cache-arc-prod": "cross-env NODE_ENV=production arcjs_network=live ts-node src/bg_cache_worker.ts",
+    "cache-arc-staging": "cross-env NODE_ENV=production ts-node src/bg_cache_worker.ts",
+    "cache-arc-prod": "cross-env NODE_ENV=production ts-node src/bg_cache_worker.ts",
     "build": "cross-env NODE_ENV=production webpack --config webpack.prod.config.js",
     "heroku-postbuild": "npm run build",
     "ganache": "npm explore @daostack/arc.js -- npm start ganache.run",
     "ganacheDb": "npm explore @daostack/arc.js -- npm start ganacheDb.run",
     "ganacheDb-clean": "npm explore @daostack/arc.js -- npm start ganacheDb.clean",
     "ganacheDb-save": "npm explore @daostack/arc.js -- npm start ganacheDb.zip",
     "ganacheDb-restore": "npm explore @daostack/arc.js -- npm start ganacheDb.restoreFromZip",
-    "migrate-ganache": "cross-env arcjs_network=ganache npm explore @daostack/arc.js -- npm start migrateContracts && npm explore @daostack/arc.js -- npm start createGenesisDao && node ./scripts/mintGen.js",
+    "migrate-ganache": "npm explore @daostack/arc.js -- npm start migrateContracts && npm explore @daostack/arc.js -- npm start createGenesisDao && node ./scripts/mintGen.js",
```

Comment:
how does this work? doesnt arc.js need the network? or I guess it defaults to ganache